designsafe/apps/projects/models/agave/base.py

```diff
@@ -487,7 +487,7 @@ def to_datacite_json(self):
         # remember, related works are not required, so they can be missing...
         for a_proj in self.associated_projects: #relatedwork
             identifier = {}
-            mapping = {'Linked Project': 'IsSupplementTo', 'Cited By': 'IsCitedBy', 'Context': 'IsDocumentedBy'}
+            mapping = {'Linked Project': 'IsSupplementTo', 'Linked Dataset': 'IsSupplementTo', 'Cited By': 'IsCitedBy', 'Context': 'IsDocumentedBy'}
```

Comment:
Keeping both the old Linked Project and the new Linked Dataset because some projects might still have Linked Project related work types and this will allow those projects to be published without our intervention